src/components/Votes.jsx

```diff
@@ -0,0 +1,32 @@
+import { updateVotes } from "../utils/api"
+import {useState} from "react"
+import "../css/Votes.css"
+
+export default function Votes({articleId, setIndividualArticle}) {
+const [error, setError] = useState(null)
+
+function handleClick(event) {
+    let increment
+    if(event.target.innerText === "Vote 👍") increment = 1
+    else{increment = -1}
+
+    const votes = {inc_votes: increment}
+    updateVotes(articleId, votes)
+    .then(() => {setError(null)})
+    .catch((error) => {
+        setIndividualArticle((currArticle) => {
+            return {...currArticle, votes: currArticle.votes - increment}
+        })  
+        setError("likes not available")
```

Comment:
Nice implementation of optimistic rendering!
Great that you've alerted the user if there's an error